app/controllers/search_controller.rb

```diff
@@ -196,6 +198,14 @@ def format_integer_params
   # nil differently. This is a helper method to remove any params that are
   # blank before passing it to Elasticsearch.
   def sanitize_params
-    params.delete_if { |_k, v| v.blank? }
+    params.compact_blank!
+    remove_invalid_sort_directions
+  end
+
+  def remove_invalid_sort_directions
+    return unless params.key?(:sort_direction)
+
+    direction = params[:sort_direction].downcase.to_sym
+    params.delete(:sort_direction) unless direction.in?(VALID_SORT_DIRECTIONS)
```

Comment:
a very valid decision would be to choose :desc here (replace instead of remove) - however there were existing nil safe initializations in some of the classes (setting a default sort order) and I decided to delegate that decision to the classes.

The argument against that is that the parameter cleaning could be a controller concern, and the internal query/fetch/search service objects expect to receive valid arguments in all situations.

There are probably a few layers of decision making here, and I didn't see a consistent pattern (which is why I added nil-safety in a few places).